web-client/integration-tests/journey/petitionerSearchesForUnassociatedSealedCase.js

```diff
@@ -0,0 +1,35 @@
+import { caseDetailHelper as caseDetailHelperComputed } from '../../src/presenter/computeds/caseDetailHelper';
+import { runCompute } from 'cerebral/test';
+import { withAppContextDecorator } from '../../src/withAppContext';
+
+const caseDetailHelper = withAppContextDecorator(caseDetailHelperComputed);
+
+export const petitionerSearchesForUnassociatedSealedCase = cerebralTest => {
+  return it('petitioner searches for unassociated sealed case', async () => {
+    // cerebralTest.setState('caseDetail', {});
+    const sealedSeedCaseDocketNumber = '105-20';
+
+    await cerebralTest.runSequence('gotoDashboardSequence');
+
+    // cerebralTest.setState('header.searchTerm', sealedSeedCaseDocketNumber);
```

Comment:
hm, i think these comments are lingering from testing and/or still helpful?

Reply:
removed-thanks